packages/@o3r/design/src/core/design-token/renderers/metadata/design-token-definition.renderers.ts

```diff
@@ -17,6 +17,30 @@ interface MetadataTokenDefinitionRendererOptions {
  * Retrieve the Design Token Variable renderer for Metadata
  * @param options
  * @returns
+ * @example Customize metadata renderer
+ * ```typescript
+ * const getCustomMetadataTokenValueRenderer = (options?: MetadataTokenValueRendererOptions): TokenValueRenderer => {
+ *   const defaultMetadataRender = getMetadataTokenValueRenderer(options);
+ *   return  (variable, variableSet) => {
+ *     const defaultMetadataObj = JSON.parse(defaultMetadataRender(variable, variableSet));
+ *     // Add custom field
+ *     defaultMetadataObj.myField = 'custom value';
+ *     return JSON.stringify(defaultMetadataObj);
+ *   };
+ * };
+ *
+ * // List of Design Token item parsed
+ * const parsedTokenDesign = await parseDesignTokenFile('./path/to/spec.json');
+ *
+ * // Renderer of the token
+ * const tokenValueRenderer = getCustomMetadataTokenValueRenderer();
+ *
+ * // Metadata Variable Renderer
```

Comment:
```suggestion
 * // Metadata variable renderer
```

Reply:
Actually I am not quite sure what is the benefit of this comment as the variable name is self-explanatory and the code is quite simple.